practice-two/src/constants/messages.js

```diff
@@ -1,14 +1,26 @@
 export const MESSAGES = {
   ERRORS: {
-    GET_API: "Having some issues on server",
-    ADD_FAILED: "Create new customer failed",
-    UPDATE_FAILED: "Update customer failed",
-    DELETE_FAILED: "Delete customer failed",
+    GET_API: 'Having some issues on server',
```

Comment:
should wrap up by GET, POST

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/996d486a-e7cf-4c60-a434-3be0f8e968b6)